src/components/select/MultiSelect.js

```diff
@@ -62,7 +62,11 @@ export default {
       type: String,
       default: 'glyphicon glyphicon-ok'
     },
-    itemSelectedClass: String
+    itemSelectedClass: String,
+    fireEvent: {
```

Comment:
I think the props is not neccessary. pls simply fire the `search` event whiout config.